src/Tribe/Tickets_View.php

```diff
@@ -1069,8 +1069,25 @@ public function get_tickets_block( $post, $echo = true ) {
 			'submit_button_name'          => $submit_button_name,
 			'cart_url'                    => method_exists( $provider, 'get_cart_url' ) ? $provider->get_cart_url() : '',
 			'checkout_url'                => method_exists( $provider, 'get_checkout_url' ) ? $provider->get_checkout_url() : '',
+			'attendees'                   => null,
+			'attendees_total'             => null,
 		];
 
+		// Check to see if attendees list is being hidden or not.
+		if ( class_exists( 'Tribe__Tickets_Plus__Attendees_List', false ) ) {
+			// Handle Event Tickets Plus compatible logic.
+			$hide_attendee_list_optout = Tribe__Tickets_Plus__Attendees_List::is_hidden_on( $post_id );
```

Comment:
We try to avoid having ET+ code in ET. What I would suggest is that we have the result of `\Tribe\Tickets\Events\Attendees_List::is_hidden_on( $post_id )` in a filter, and hook from ET+ to alter the value when needed.

Reply:
That makes sense. I cloned the logic from the RSVP template https://github.com/the-events-calendar/event-tickets/blob/master/src/Tribe/RSVP.php#L391 should I take care of both with this PR?